api/tests/test_cloud_record_summary.py

```diff
@@ -18,8 +15,134 @@ class CloudRecordSummaryTest(TestCase):
     """Tests GET requests to the Cloud Sumamry Record endpoint."""
 
     def setUp(self):
-        """Disable logging.INFO from appearing in test output."""
-        logging.disable(logging.INFO)
+        """Prevent logging from appearing in test output."""
+        logging.disable(logging.CRITICAL)
+
+    def test_cloud_record_summary_get_IAM_fail(self):
+        """
+        Test what happens if we fail to contact the IAM.
+
+        i.e, _token_to_id returns None
+        """
+        CloudRecordSummaryView._token_to_id = Mock(return_value=None)
```

Comment:
Thanks for the extra comment though there are a few other places similar mocks are used which could do with their comments being clarified the same way.